app/controllers/organizations_controller.rb

```diff
@@ -9,6 +9,7 @@ def create
     authorize @organization
     if @organization.save
       current_user.update(organization_id: @organization.id, org_admin: true)
```

Comment:
Probably, `update` and `create` only make sense together, so I suggest wrapping them in a transaction.